### PR TITLE
Environment B & Service 1 - fargate-env & load-balanced-fargate-svc

### PR DESCRIPTION
**Pull request by jona62**
Adds terraform template for;
fargate environment
loadbalanced fargate service with pipeline

### CODE REVIEW

File: service-templates/load-balanced-fargate-svc/v1/instance_infrastructure/output.tf [hunk 1]
```diff
@@ -0,0 +1,3 @@
+output "ServiceURL" {
```
Comment (mikewrighton) [inline service-templates/load-balanced-fargate-svc/v1/instance_infrastructure/output.tf:1]: URL -> Url

File: service-templates/load-balanced-fargate-svc/v1/pipeline_infrastructure/config.tf [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
+  }
+
+  backend "s3" {
+    region = "us-east-1"
+    bucket = "terraform-states-858487653465"
+    key    = "fargate-pipeline"
+  }
```
Comment (mikewrighton) [inline service-templates/load-balanced-fargate-svc/v1/pipeline_infrastructure/config.tf:13]: We should leave the backend block empty for Proton.

File: service-templates/load-balanced-fargate-svc/v1/pipeline_infrastructure/outputs.tf [hunk 3]
```diff
@@ -0,0 +1,3 @@
+output "pipeline_endpoint" {
```
Comment (mikewrighton) [inline service-templates/load-balanced-fargate-svc/v1/pipeline_infrastructure/outputs.tf:1]: pipeline_endpoint -> PipelineEndpoint

File: service-templates/load-balanced-fargate-svc/v1/instance_infrastructure/output.tf (hunk 1 above)
Comment (jona62) [reply service-templates/load-balanced-fargate-svc/v1/instance_infrastructure/output.tf:1]: Done

File: service-templates/load-balanced-fargate-svc/v1/pipeline_infrastructure/outputs.tf (hunk 3 above)
Comment (jona62) [reply service-templates/load-balanced-fargate-svc/v1/pipeline_infrastructure/outputs.tf:1]: Done

**Pull request closed without merging**